recipe/meta.yaml

```diff
@@ -181,6 +182,64 @@ outputs:
         Compiler for CUDA applications.
       doc_url: https://docs.nvidia.com/cuda/index.html
 
+  - name: cuda-crt-tools
+    files:       # [linux]
+      - bin/crt  # [linux]
+    requirements:
+      host:
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+        - cuda-version {{ cuda_version }}
+      run:
+        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+      run_constrained:
+        - arm-variant * {{ arm_variant_type }}    # [aarch64]
+    test:
+      commands:
+        - test -d $PREFIX/bin/crt            # [linux]
+        - test -f $PREFIX/bin/crt/link.stub  # [linux]
+    about:
+      home: https://developer.nvidia.com/cuda-toolkit
+      license_file: LICENSE
+      license: LicenseRef-NVIDIA-End-User-License-Agreement
+      license_url: https://docs.nvidia.com/cuda/eula/index.html
+      summary: Compiler for CUDA applications.
+      description: |
+        Compiler for CUDA applications.
+      doc_url: https://docs.nvidia.com/cuda/index.html
+
+  - name: cuda-crt_{{ target_platform }}
```

Comment:
Would it make sense to call this a dev package like we did with `cuda-cudart-dev`

```suggestion
  - name: cuda-crt-dev_{{ target_platform }}
```

Reply:
Done in https://github.com/conda-forge/cuda-nvcc-impl-feedstock/pull/8/commits/454ce06881ec73c859d70dce13872a81d2431978